tests/unit/MigrationTest.ts

```diff
@@ -87,12 +90,13 @@ describe('Migrations', () => {
                                     previousReportActionID: '1',
                                 },
                             };
-                            expect(allReportActions[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}1`]).toMatchObject(expectedReportAction);
+                            expect(allReportActions?.[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}1`]).toMatchObject(expectedReportAction);
                         },
                     });
                 }));
 
         it('Should skip zombie report actions and proceed to remove all reportActions given that a previousReportActionID does not exist', () =>
+            // @ts-expect-error Preset necessary values
             Onyx.multiSet({
```

Comment:
Please have a look [at this discussion on Slack](https://callstack-hq.slack.com/archives/C05LWS6EL3E/p1709223334828259).